serrano/resources/preview.py

```diff
@@ -67,11 +68,21 @@ def get(self, request):
         # depend on data to exist!
         header = []
         ordering = OrderedDict(view_node.ordering)
+        concepts = view_node.get_concepts_for_select()
+
+        # Prepare an HTMLExporter
+        exporter = processor.get_exporter(HTMLExporter)
+
+        objects = []
+        header = []
+
+        # Skip pk field.
+        for i, f in enumerate(exporter.header[1:]):
```

Comment:
I might be missing this but where is the `id` field explicitly set as the first one in the header? Should we enumerate over all the fields and check if the field is the `id` within each loop iteration?